app/build.gradle

```
@@ -38,6 +38,7 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
+    implementation 'com.google.firebase:firebase-firestore:21.7.1'
```

Also yes we could simplify this to 
```suggestion
    implementation 'com.google.firebase:firebase-firestore'
```

---

Good catch. Turns out we already had firestore imported somewhere else too.